internal/api.go

```diff
@@ -43,6 +37,38 @@ func NewAPI(config api.Config) (api.API, error) {
 	}, nil
 }
 
+func loadRules(conversionRulesPath string) (RuleSet, error) {
+	ruleSet := RuleSet{
+		rules: []Rule{},
+	}
+
+	filenames, err := filepath.Glob(filepath.Join(conversionRulesPath, "*.yaml"))
+	if err != nil {
+		return RuleSet{}, err
+	}
+
+	for _, filename := range filenames {
+		file, err := os.Open(filename)
```

Comment:
It's idiomatic to check for an error opening the file, and then defer closing the file.

For reference: https://gobyexample.com/writing-files